bf2fc6cc711aee1a0c2a/kas-fleet-manager
# fix: ensure the name is set for the subscription resource

## Description


This PR makes sure the package name is set on the Subscription resource for the managed-kafka addon so that the install plan will  be set to `Manual` by default.

## Verification Steps


* Make sure you have an existing cluster running, with the Strimzi addon installed. The install plan should be "Automatic"
* Run this PR locally and point it to the OSD cluster, make sure it is set to ready in the db. After the cluster reconciler runs you should see the install plane for the Strimzi addon is set to manual.

## Type of change


## Checklist (Definition of Done)
